src/java/org/apache/cassandra/config/Config.java

```diff
@@ -171,6 +171,20 @@ public static Set<String> splitCommaDelimited(String src)
 
     public volatile DurationSpec.LongMillisecondsBound stream_transfer_task_timeout = new DurationSpec.LongMillisecondsBound("12h");
 
+    public volatile DurationSpec.LongMillisecondsBound cms_await_timeout = new DurationSpec.LongMillisecondsBound("120000ms");
+    public volatile int cms_default_max_retries = 10;
+    public volatile DurationSpec.IntMillisecondsBound cms_default_retry_backoff = new DurationSpec.IntMillisecondsBound("50ms");
```

Comment:
I think we can remove the "default_" part, no? It is a configurable parameter whose default value is set, no? Same for `cms_default_max_retries`

Reply:
A quick look shows me we have diverged from using `initial_` to `default_` in the past year in the `Config` class. Now, we are inconsistent. The precedent was not created in this patch; sadly, as it could be confusing to people, it happened. I leave it up to you; naming is hard in general.